src/Apps/Auction/Routes/__tests__/ConfirmBid.test.tsx

```diff
@@ -124,6 +132,8 @@ const setupTestEnv = ({
 
 describe("Routes/ConfirmBid", () => {
   beforeEach(() => {
+    ;(sd as any).ENABLE_PRICE_TRANSPARENCY = false
```

Comment:
Similar comment to above. Once we add the mapping of `ENABLE_PRICE_TRANSPARENCY` to the `GlobalData` type, we might not need this `as any`.

https://github.com/artsy/reaction/blob/master/typings/sharify.d.ts

Reply:
I've also updated the tests as part of https://github.com/artsy/reaction/pull/3033/commits/4a5361fd563dc066cc0f74b3da60791e19a851d5 since now it's a `readonly` field that can't be mutated, and mutating it isn't good anyway, so I just switched to a jest mock.